tests/foreman/ui/test_host.py

```diff
@@ -2466,3 +2466,134 @@ def test_positive_set_multi_line_and_with_spaces_parameter_value(
         assert host_parameters
         assert param_name in host_parameters
         assert host_parameters[param_name] == param_value
+
+
+class TestHostAnsible:
+    """Tests for Ansible portion of Hosts"""
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier2
+    def test_positive_host_role_information(self):
+        """Assign Ansible Role to a Host and an attached Host group and verify that the information
+        in the new UI is displayed correctly
+
+        :id: 7da913ef-3b43-4bfa-9a45-d895431c8b56
+
+        :assignee: sbible
+
+        :Steps:
+            1. Register a RHEL host to Satellite.
+            2. Import all roles available by default.
+            3. Create a host group and assign one of the Ansible roles to the host group.
+            4. Assign the host to the host group
+            5. Assign one roles to the RHEL host.
+            6. Navigate to the new UI for the given Host
+            7. Select the 'Ansible' tab, then the 'Inventory' sub-tab.
+
+        :expectedresults: Roles assigned directly to the Host are visible on the subtab, and
+        roles assigned to the Host Group are visible by clicking the "view all assigned roles" link
+        """
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier2
+    def test_positive_role_variable_information(self):
+        """Create and assign variables to an Ansible Role and verify that the information in
+        the new UI is displayed correctly
+
+        :id: 4ab2813a-6b83-4907-b104-0473465814f5
+
+        :assignee: sbible
+
+        :Steps:
+            1. Register a RHEL host to Satellite.
+            2. Import all roles available by default.
+            3. Create a host group and assign one of the Ansible roles to the host group.
+            4. Assign the host to the host group
+            5. Assign one roles to the RHEL host.
+            6. Create a variable and associate it with the role assigned to the Host
+            7. Create a variable and associate it with the role assigned to the Hostgroup
+            8.  Navigate to the new UI for the given Host
```

Comment:
Looks like there's an extra space after `8.` on this line and `9.` on the following line.